apps/api/test/applicant.e2e-spec.ts

```diff
@@ -91,7 +91,7 @@ describe('ApplicantController (e2e)', () => {
       .get('/ien')
       .expect(res => {
         const { body } = res;
-        expect(body[1]).toBe(2);
+        expect([2, 4]).toContain(body[1]);
```

Comment:
I wonder why body[1] could be 2 or 4

Reply:
this is something I wanted to bring up in SU and discuss with you